generators/client/templates/react/src/main/webapp/app/reducers/user-management.ts

```diff
@@ -100,10 +102,13 @@ export default (state = initialState, action) => {
 
 const apiUrl = '/api/users';
 // Actions
-export const getUsers: ICrudGetAction = (page, size, sort) => ({
-  type: ACTION_TYPES.FETCH_USERS,
-  payload: axios.get(`${apiUrl}?cacheBuster=${new Date().getTime()}`)
-});
+export const getUsers: ICrudGetAction = (page, size, sort) => {
+  const requestUrl = `${apiUrl}` + (sort ? `?page=${page}&size=${size}&sort=${sort}` : ``);
```

Comment:
you can also do `` `${apiUrl}${sort ? `?page=${page}&size=${size}&sort=${sort}` : ''}`;``